app/assets/javascripts/cascade/level/rotator.js

```diff
@@ -31,7 +31,7 @@ $(function () {
         var beforeSlideChange = function(slider) {
             if ( !isSliderMasthead ) { return; }
 
-            $header.fadeOut();
```

Comment:
Oh I see what you're saying. By jump I meant that the red header block would completely disappear and reappear on slide change. The height jump that is currently happening is because some slides have more text and the height needs to adjust. Wai said that is fine as there is not really a way around that.